docs/en/latest/config.json

```diff
@@ -6,18 +6,25 @@
       "label": "Architecture Design",
       "items": [
         "architecture-design/apisix",
-        "architecture-design/route",
-        "architecture-design/plugin",
-        "architecture-design/script",
-        "architecture-design/service",
-        "architecture-design/consumer",
-        "architecture-design/upstream",
-        "architecture-design/global-rule",
         "architecture-design/plugin-config",
-        "architecture-design/router",
         "architecture-design/debug-mode"
       ]
     },
+    {
+      "type": "category",
+      "label": "Terminology",
+      "items": [
+        "terminology/api-gateway",
```

Comment:
cc @membphis, if we need to adjust those items' order?

Reply:
I have put them in alphabetical order, as we do with dictionaries since terminologies are similar to dictionaries. Please let me know if anything else is more accurate.

Reply:
How about this way:

1. API Gateway
2. Consumer
3. Upstream
4. Route
5. Service
6. xxx

Reply:
this order is fine for me:

API Gateway
Route
Service
Upstream
Consumer
xxx

Reply:
Okay, I will change it.